next-app/src/app/layout.tsx

```diff
@@ -7,6 +7,14 @@ export default function RootLayout({
 }) {
   return (
     <html lang="en">
+      <head>
+        <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.3/dist/leaflet.css"
```

Comment:
aqui eu vejo uma boa oportunidade de talvez utilizar mais o webpack talvez
direct import assim normalmente não é um padrão muito bom, acabamos importando muita coisa que nao é utilizado

talvez esse artigo de uma luz https://www.digitalocean.com/community/tutorials/how-to-dynamically-import-javascript-with-import-maps#step-3-loading-external-code-with-import-maps

Reply:
Eu adicionei quando tava testando o leaflet, nem precisei no final